docs/cisco.ios.ios_acls_module.rst

```diff
@@ -3876,11 +3876,14 @@ Parameters
                 </td>
                 <td>
                         <div>The state the configuration should be left in</div>
-                        <div>The states <em>merged</em> is the default state which merges the want and have config, but for ACL module as the IOS platform doesn&#x27;t allow update of ACE over an pre-existing ACE sequence in ACL, same way ACLs resource module will error out for respective scenario and only addition of new ACE over new sequence will be allowed with merge state.</div>
+                        <div>The state <em>merged</em> is the default state which merges the want and have config, but for ACL module as the IOS platform doesn&#x27;t allow update of ACE over an pre-existing ACE sequence in ACL, same way ACLs resource module will error out for respective scenario and only addition of new ACE over new sequence will be allowed with merge state.</div>
```

Comment:
@tin-ot this should not be a manual edit, would you please use https://github.com/ansible-network/collection_prep and make changes to the documentation and use this collection to generate the .rst for you